runtime/model-protobuf/src/test/java/io/aklivity/zilla/runtime/model/protobuf/internal/ProtobufParserTest.java

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright 2021-2024 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.model.protobuf.internal;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import com.google.protobuf.Descriptors.FileDescriptor;
+import com.google.protobuf.Descriptors.Descriptor;
+import com.google.protobuf.Descriptors.FieldDescriptor;
+
+
+public class ProtobufParserTest
+{
+    @Test
+    public void shouldParseProto2Schema()
+    {
+        // Given: A proto2 schema
+        String proto2Schema =
+                "syntax = \"proto2\";\n" +
+                        "package example;\n" +
+                        "\n" +
+                        "message Person {\n" +
+                        "  required string name = 1;\n" +
+                        "  optional int32 age = 2;\n" +
+                        "  repeated string email = 3;\n" +
+                        "}\n";
```

Comment:
Please use triple-quote multi-line strings instead of separate strings with +.
Applies to all similar multi-line strings.